xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.css

```diff
@@ -147,11 +147,14 @@ ul.commentreplies, .reply {
   /* fix for IE7 */
   width: 98%;
 }
-/* Permalink box */
-.permalinkBox .xdialog-box-confirmation .question {
+/* Permalink modal */
+#permalinkModal .input-group-addon{
   background-image: url("$xwiki.getSkinFile('icons/silk/link.png')");
```

Comment:
I prefer the icon to be set using the Icon Themes, from the structure, instead as background-color.